### PR TITLE
[SECURITY] Protect frame GET parameter in tx_cms_showpic eID

### PR DESCRIPTION
**Pull request by linawolf**
releases: main, 12.4, 11.5
resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/917

### CODE REVIEW

File: Documentation/Configuration/Typo3ConfVars/SYS.rst [hunk 1]
```diff
@@ -658,6 +658,33 @@ configurations.
...
+
+    ..  confval:: security.frontend.allowInsecureFrameOptionInShowImageController
+        :name: globals-typo3-conf-vars-sys-features-security-frontend-aallowInsecureFrameOptionInShowImageController
+        :Path: $GLOBALS['TYPO3_CONF_VARS']['SYS']['features']['security.frontend.allowInsecureFrameOptionInShowImageController']
+        :type: bool
+        :Default: false
+
+        ..  versionadded:: 13.1, 12.4.15, 11.5.37
+
+        The show image controller (eID `tx_cms_showpic`) lacks a cryptographic
```
Comment (garvinhicking) [inline Documentation/Configuration/Typo3ConfVars/SYS.rst:671]: Don't have the time now but I think we should rephrase that less changelogy and much shorter:

To restore behavior of older TYPO3 versions, this parameter can be enabled to allow non-cryptogrpahically signed URLs (lacking 'cHash'). It is strongly suggested to leave it turned off (default).

Maybe a "changelog entry xxx reveals reasoning for this" reference?

Comment (linawolf) [reply Documentation/Configuration/Typo3ConfVars/SYS.rst:671]: Why dont we merge it like this for now and you improve the text in a follow up once you have time? At least we have the option mentioned then

Comment (garvinhicking) [reply Documentation/Configuration/Typo3ConfVars/SYS.rst]: I now made a proposal to change it, is that ok for you @linawolf ?

**Comment by linawolf** [comment]
Thanks, shall we merge then?

**Review by garvinhicking — APPROVED** [review]

**Comment by garvinhicking** [comment]
@linawolf Merged and did the 12.4 backport but 11.5 seems harder because the whole layout changed there

**Pull request merged**